test-case/check-userspace-cardinfo.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+##
+## Case Name: check-userspace-cardinfo
+## Preconditions:
+##    N/A
+## Description:
+##    Get the card name from pactl info
+##    check if there is an available card to do the following tests
+## Case step:
+##    1. run pactl to get the cards info
+##    2. check the card name
+## Expect result:
+##    There is at least one available card for test
+##
+
+# shellcheck source=case-lib/lib.sh
+source "$(dirname "${BASH_SOURCE[0]}")"/../case-lib/lib.sh
+
+func_opt_parse_option "$@"
+
+OLDIFS=$IFS
+cardlist=$(pactl list cards short)
```

Comment:
Can you add some env check for pactl first? I think the pulseaudio need to be active for this cmd to be run

Reply:
@xiulipan This test case runs after check-userspace-pa-status. In check-userspace-pa-status, it will check pulseaudio is started correctly or not.

Reply:
@libinyang I think `check-userspace-pa-status` should be a lib instead of a test case.
Please also consider about single test case run situation.

Reply:
> A simple question here. If we have a help function, why still need `|` and jq for the string process? Why not do something like `sof-dump-status.py` to use args to directly output info you need.

@xiulipan This is a good question. It's really a hard decision. Finally I decide to make the pactlinfo.py be a simple tool and let the logical analysis in the test case. This is because:
1. The similar function in  apport is “parse_cards()" and it seems it will call parse_cards() when it needs the information and don't have help function either. What's more:
2. We may need many informations from alsactl. If we use helper function, we may need a lot of such functions when we have more and more test cases later.

Both methods have pros and cons. I'm open for the final solution.